lib/index.js

```diff
@@ -155,6 +157,8 @@ module.exports = function(options) {
 
   options.batchSize = options.batchSize || 25;
   options.batchMaxTime = options.batchMaxTime || 2000;
+  options.requestMaxBodySizeLimit = options.requestMaxBodySizeLimit || 100000;
+  options.responseMaxBodySizeLimit = options.responseMaxBodySizeLimit || 100000;
```

Comment:
Do we need to call it both "max" and "limit"? Seems redundant. XxMadBodySize seems sufficient